conftest.py

```diff
@@ -110,12 +123,10 @@ def pytest_sessionstart(session):
         device = "gaudi2" if os.environ["GAUDI2_CI"] == "1" else "gaudi1"
     # Try to automatically detect it
     else:
-        import habana_frameworks.torch.hpu as torch_hpu
-
-        name = torch_hpu.get_device_name().strip()
+        name = get_device_name_in_separate_process()
         if not name:
             raise RuntimeError("Expected a Gaudi device but did not detect one.")
-        device = name.split()[-1].lower()
+        device = name.strip().split()[-1].lower()
```

Comment:
Change this block to:
```python
from optimum.habana.utils import get_device_name
device = get_device_name()
````

Reply:
get_device_name already throws exception if device is unrecognized, so we don't need to do it here anymore.